index.js

```diff
@@ -70,5 +70,11 @@ module.exports = {
     ],
 
     '@typescript-eslint/no-explicit-any': ['error'],
+
+    /** Misc */
+
+    // Fix no shadow error on enum, see https://github.com/typescript-eslint/typescript-eslint/issues/2483
+    'no-shadow': 'off',
+    '@typescript-eslint/no-shadow': 'error',
```

Comment:
C'est chiant quand même car cette règle est bien à la base

Reply:
The rule is still activated, just use the typescript one because the js 'no-shadow' cause errors on enum see 'no-await-in-loop'